ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -110,9 +110,18 @@ tag_specification:
 tests:
 - as: verify
   cluster: build02
-  commands: make verify
-  container:
-    from: src
+  steps:
+    test:
+    - as: verify
+      commands: make verify
+      env:
+      - default: /tmp/
+        name: GOLANGCI_LINT_CACHE
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```

Comment:
My only nit, is this enough for good scheduling? We can always check after a few executions to adjust.

Reply:
It's the same as the global values we set for all tests, so it's probably fine. For whatever reason, steps under the test stanza don't inherit the global request values, according to the linter anyway.